libraries/AP_HAL_F4BY/AP_HAL_F4BY_Namespace.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace F4BY {
+	class F4BYScheduler;
+	class F4BYUARTDriver;
+	class F4BYStorage;
+	class F4BYRCInput;
+	class F4BYRCOutput;
+	class F4BYAnalogIn;
+	class F4BYAnalogSource;
+	class F4BYUtil;
+        class F4BYGPIO;
+        class F4BYDigitalSource;
+        class NSHShellStream;
+        class F4BYI2CDriver;
+        class F4BY_I2C;
+        class Semaphore;
+}
```

Comment:
Declaring classes this way isn't really the best approach. See https://github.com/ArduPilot/ardupilot/pull/4574